flow/project.py

```diff
@@ -310,13 +324,19 @@ def __init__(self, id, name, job, cmd, directives=None):
         self.directives._keys_set_by_user = keys_set_by_user
 
     def __str__(self):
-        return "{}({})".format(self.name, self.job)
+        if len(self._jobs) == 1:
+            return "{}({})".format(self.name, str(self._jobs[0]))
+        else:
+            minimum = self._jobs[0]._project.min_len_unique_id()
+            return "{}-{}: ({}-{})".format(
+                self.name, len(self._jobs), str(self._jobs[0])[0:minimum],
+                str(self._jobs[-1])[0:minimum])
 
     def __repr__(self):
-        return "{type}(name='{name}', job='{job}', cmd={cmd}, directives={directives})".format(
+        return "{type}(name='{name}', jobs='{jobs}', cmd={cmd}, directives={directives})".format(
                    type=type(self).__name__,
                    name=self.name,
-                   job=str(self.job),
+                   jobs="[" + " ,".join(map(str, self._jobs)) + "]",
```

Comment:
This is also changing behavior, it is fine since this will be mostly internal, just marking it, since we will need to change it for `JobOperation` not `_JobOperation` per #325

Reply:
Does this even work? The `__repr__` return value should be able to be used in `eval()` like this: `new_foo == eval(repr(foo))`.

Reply:
This should possibly get resolved by using `map(repr, self._jobs)`